<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="5">
    <widget class="QPushButton" name="AboutBtn">
     <property name="text">
      <string>关于</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="svrsBtn">
     <property name="text">
      <string>结果另存为...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>另存为...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="fileStatus">
     <property name="text">
      <string>未打开文件</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="openFileBtn">
     <property name="text">
      <string>打开...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>作压缩处理</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="scanStatus">
     <property name="text">
      <string>未分析</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTextBrowser" name="codeView"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="scanBtn">
     <property name="text">
      <string>词法分析</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QTableWidget" name="resultView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
